packages/core/saved-objects/docs/openapi/bundled.yaml

```diff
@@ -18,6 +18,151 @@ tags:
   - name: saved objects
     description: Manage Kibana saved objects, including dashboards, visualizations, and more.
 paths:
+  /api/saved_objects/_export:
+    post:
+      summary: Retrieve sets of saved objects that you want to import into Kibana.
+      operationId: exportSavedObjects
+      description: |
+        This functionality is in technical preview and may be changed or removed in a future release. Elastic will apply best effort to fix any issues, but features in technical preview are not subject to the support SLA of official GA features. NOTE: The `savedObjects.maxImportExportSize` configuration setting limits the number of saved objects which may be exported.
+      tags:
+        - saved objects
+      parameters:
+        - $ref: '#/components/parameters/kbn_xsrf'
+      requestBody:
```

Comment:
I'm not super familiar with reading these docs and I know it was mentioned that they will look different later on, but is there extraneous data showing up here? What is the `Any of` on the bottom?
<img width="401" alt="Screenshot 2023-08-23 at 1 00 27 PM" src="https://github.com/elastic/kibana/assets/21210601/6055f087-21f9-4846-a4b2-3eb35a778913">

Reply:
Thanks for catching this. I've got this fixed up with [`ce0a927` (#162566)](https://github.com/elastic/kibana/pull/162566/commits/ce0a92711355a47c47a505f1a9b4b97791ee7999)

Here's an updated screenshot:

<img width="407" alt="Screenshot 2023-08-23 at 5 15 21 PM" src="https://github.com/elastic/kibana/assets/40268737/fd1d0b45-8c40-4594-8234-42b319905a8a">